cmake/CMakeLists.txt

```diff
@@ -1291,6 +1291,11 @@ if (onnxruntime_ENABLE_EXTENSION_CUSTOM_OPS)
   set(OCOS_ENABLE_CTEST OFF CACHE INTERNAL "")
   set(OCOS_ENABLE_STATIC_LIB ON CACHE INTERNAL "")
   set(OCOS_ENABLE_SPM_TOKENIZER OFF CACHE INTERNAL "")
+
+  if (onnxruntime_EXTENSION_CUSTOM_OPS_SELECTED)
+    set(OCOS_ENABLE_SELECTED_OPLIST ON CACHE INTERNAL "")
```

Comment:
Can we directly set up this flag in the build.py?

Reply:
Thanks for the comment. I don't think this option `OCOS_ENABLE_SELECTED_OPLIST` should be set in build.py because build.py should set cmake options starting with `-Donnxruntime_` and common options like `'-A', 'ARM'`. It should not set feature options directly.
And the options which would be passed to our onnxruntime-extensions should all be set here. This would make the cmake options consistent and clear to maintain.
Please let me know if you have different ideas. Thanks.